src/Foundation.Find.Cms/Config/IgnoreCollectionEditorDescriptor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using EPiServer.Cms.Shell.UI.ObjectEditing.EditorDescriptors;
+using EPiServer.DataAnnotations;
+using EPiServer.Shell.ObjectEditing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done